src/app/clusters/bindings/BindingManager.h

```diff
@@ -93,6 +96,11 @@ class BindingManager
      */
     CHIP_ERROR NotifyBoundClusterChanged(EndpointId endpoint, ClusterId cluster, void * context);
 
+    /*
+     * Notify the BindingAddedHandler that a binding was added.
+     */
+    CHIP_ERROR NotifyBindingAdded(const EmberBindingTableEntry & binding);
```

Comment:
Why do we want to do this via the BindingManager instead of directly registering this callback with the binding cluster?

And why do we think a single callback recipient is enough?

It might be better to just have a "bindings modified" event dispatched via the event system and then consumers can act on that as desired...

Reply:
It's helpful for the handler to receive the EmberBindingTableEntry object. What do you mean by "registering this callback with the binding cluster" - are you referring to the BindingTableAccess class?

can you point me to an example in the code where an event dispatched via the event system?

Reply:
added code to return an error if more than one client attempts to register a callback. I can make this more flexible if the need arises.

Reply:
> can you point me to an example in the code where an event dispatched via the event system?

`DeviceControlServer::CommissioningComplete` has an example.  But we can do that as a followup....

Reply:
> are you referring to the BindingTableAccess class?

I was referring to having the binding table expose some API to register change listeners, yes.  BindingManager is not that; it's the thing that manages actual CASE connections to bindings....